src/components/ProductVersionMilestonesList/ProductVersionMilestonesList.tsx

```diff
@@ -0,0 +1,103 @@
+import { ActionsColumn, TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import { ProductMilestone } from 'pnc-api-types-ts';
+
+import { IServiceContainer } from 'hooks/useServiceContainer';
+
+import { ContentBox } from 'components/ContentBox/ContentBox';
+import { Filtering, IFilterAttributes } from 'components/Filtering/Filtering';
+import { Pagination } from 'components/Pagination/Pagination';
+import { ProductMilestoneReleaseLabel } from 'components/ProductMilestoneReleaseLabel/ProductMilestoneReleaseLabel';
+import { useServiceContainerProductVersion } from 'components/ProductVersionPages/ProductVersionPages';
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+import { Toolbar } from 'components/Toolbar/Toolbar';
+import { ToolbarItem } from 'components/Toolbar/ToolbarItem';
+
+import { createDateTime } from 'utils/utils';
+
+const filterAttributes: IFilterAttributes = {
+  filterAttributes: {
+    version: {
+      id: 'version',
+      title: 'Name',
+      operator: '=like=',
+    },
+  },
+};
+
+interface IProductVersionMilestonesListProps {
+  serviceContainerProductMilestones: IServiceContainer;
+  componentId: string;
+}
+
+/**
+ * Component displaying list of Product Milestones.
+ *
+ * @param serviceContainerProductMilestones - Service Container for Product Milestones
+ * @param componentId - Component ID
+ */
+export const ProductVersionMilestonesList = ({
+  serviceContainerProductMilestones,
+  componentId,
+}: IProductVersionMilestonesListProps) => {
+  const { serviceContainerProductVersion } = useServiceContainerProductVersion();
+
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem>
+          <Filtering filterAttributes={filterAttributes} componentId={componentId} />
+        </ToolbarItem>
+      </Toolbar>
+
+      <ContentBox borderTop>
+        <ServiceContainerLoading {...serviceContainerProductMilestones} title="Product Milestones">
+          <TableComposable isStriped variant="compact">
+            <Thead>
+              <Tr>
+                <Th width={25}>Name</Th>
```

Comment:
Display following properties: 
 - Version (originally Name)
 - Status
 - Start Date
 - Planned End Date
 - End Date
 - Last Close Result

Reply:
It's not possible to infer the Latest Close Result from `/product-versions/:ID/milestones` endpoint.

Reply:
done